.github/workflows/build.yml

```diff
@@ -174,6 +174,11 @@ jobs:
     env:
       SKIP_INTEGRATION_TESTS: 1
     steps:
+      -
+        name: Setup Git config
+        run: |
+          git config --global core.autocrlf false
+          git config --global core.eol lf
```

Comment:
Ok seems on Windows runner eol is set to CRLF by default so needs this step before checking out.